File: 2756-buy-two-chocolates/2756-buy-two-chocolates.py
class Solution:
    def buyChoco(self, prices: List[int], money: int) -> int:
        
        minMon = -1

        n = len(prices)

        for i in range(n-1):
            for j in range(i+1, n):
                diff = money - (prices[i] + prices[j])
                if diff >= 0 and diff > minMon:
                    minMon = diff
        
        return minMon if minMon > -1 else money

File: 2756-buy-two-chocolates/2756-buy-two-chocolates.java
class Solution {
    public int buyChoco(int[] prices, int money) {
        int minMoney = -1;
        int n = prices.length;

        for (int i = 0; i < n-1; i++){
            for (int j = i+1; j < n; j++){
                int diff = money - prices[i] - prices [j];
                if (diff >= 0 && diff > minMoney){
                    minMoney = diff;
                }
            }
        }

        if (minMoney == -1){
            return money;
        } 
        else{
            return minMoney;
        }
    }
}